src/server/pfs/s3/bucket.go

```diff
@@ -146,6 +170,56 @@ func (c *controller) ListObjects(r *http.Request, bucketName, prefix, marker, de
 	return &result, err
 }
 
+func (lfs *localFS) listObjects(pc *client.APIClient, bucket *Bucket, prefix, marker string, recursive bool, maxKeys int) (*s2.ListObjectsResult, error) {
+	result := s2.ListObjectsResult{
+		Contents:       []*s2.Contents{},
```

Comment:
I think I just copied what was already there. We seem to use `[]type{}` and `&type{}` rather than `make` and `new`, which I think is the opposite of what go style recommends. Happy to flip things over in this file, though

Reply:
Staying consistent for this makes sense, even if it's not idiomatic